pallets/staking/src/lib.rs

```diff
@@ -2345,24 +2525,28 @@ impl<T: Trait> Module<T> {
 
     /// Actually make a payment to a staker. This uses the currency's reward function
     /// to pay the right payee for the given staker account.
-    fn make_payout(stash: &T::AccountId, amount: BalanceOf<T>) -> Option<PositiveImbalanceOf<T>> {
+    fn make_payout(stash: &T::AccountId, amount: Balance) -> Option<PositiveImbalanceOf<T>> {
         let dest = Self::payee(stash);
+        let native_liquidity_token = T::NativeCurrencyId::get();
```

Comment:
`native_liquidity_token` should be renamed into `native_token`, since Mangata token is not a liquidity token

Reply:
done. ty for catching that.